src/js/bootstrap-slider.js

```diff
@@ -1565,9 +1565,12 @@ const windowIsDefined = (typeof window === "object");
 				}
 
 				var val = this._state.value[handleIdx] + dir * this.options.step;
+				var percentage = (val / this.options.max) * 100;
+				this._state.keyCtrl = handleIdx;
 				if (this.options.range) {
-					val = [ (!handleIdx) ? val : this._state.value[0],
-						    ( handleIdx) ? val : this._state.value[1]];
+					this._adjustPercentageForRangeSliders(percentage);
+					val = [ (!this._state.keyCtrl) ? val : this._state.value[0],
```

Comment:
could you break these array values into separate variables and then reference them within the array?

```js
const val1 = (!this._state.keyCtrl) ? val : this._state.value[0];
const val2 = this._state.keyCtrl) ? val : this._state.value[1];

val = [ val1, val2];
```